miss_islington/tasks.py

```diff
@@ -21,6 +21,14 @@
 
 cache = cachetools.LRUCache(maxsize=500)
 
+CHERRY_PICKER_CONFIG = {
+    "team": "python",
+    "repo": "cpython",
+    "check_sha": "7f777ed95a19224294949e1b4ce56bbffcb1fe9f",
```

Comment:
One small question: what is this SHA for?

Reply:
https://github.com/python/cherry-picker/blob/39e8d6d10eecae6b6d096e3e03b69c5104ba5507/cherry_picker/cherry_picker.py#L490-L493

We needed a way to detect whether cherry-picker is working on the right git repo. We use this to detect that we are indeed in CPython repo.  If not mistaken, I think this SHA is the first commit to CPython.

Reply:
Thanks for the clarification!